tests/test_joysticks.py

```diff
@@ -0,0 +1,81 @@
+"""Test module for `joysticks.py`"""
+
+import unittest
+import unittest.mock
+
+import joysticks
+
+
+class TestJoysticks(unittest.TestCase):
+    """Test class for `Joysticks`"""
+
+    def test_get_num_joysticks(self):
+        """Test whether `Joysticks.get_num_joysticks` reports
+         the correct number of joysticks
+        """
```

Comment:
Function documentation on even the test functions ... really good!

/me wipes away a tear of joy.